girder/models/user.py

```diff
@@ -46,6 +47,11 @@ def initialize(self):
         self.exposeFields(level=AccessType.ADMIN, fields=(
             'size', 'email', 'groups', 'groupInvites'))
 
+        events.bind('model.user.save.created', 'grantSelfAccess',
```

Comment:
As I mentioned on the call today, my thinking is that it may be desirable to not always have users with admin on themselves. I could imaging a tightly managed Girder instance where users could be created via a central authority and are not allowed to change their own name or email address.

With that in mind, it no longer becomes an intrinsic precondition that users must have admin access on themselves, so we are no longer forced to do that operation inside ``createUser`` (and indeed we shouldn't, in case someone wants to unbind that event). Thus, I think moving ``setUserAccess`` to a post-create callback works quite elegantly.